src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
@zebmason got it. I think probably the most likely way to take such changes would be for you to open an issue pointing at your tool. We can have a list of libraries with a checkbox for each, simliar to https://github.com/dotnet/corefx/issues/17905. Then anyone interested can try it out and clean up a PR for a library. 

If your tool could read the existing ILLinkTrim.xml files that would make it easier as those could be used to help avoid false positives. if it makes sense, flags on the tool might make it easier to select modes that are more "reliable"

Speaking for the full time contributors on this project, we'd certainly like to remove dead code where it exists but after the recent cleanup of the worst cases it's not something we're likely to get to anytime soon. So setting up an issue for community folk to look at if interested would be best. They are likely to start there rather than volunteer to clean up a PR from someone else.

Does that sound reasonable?